### PR TITLE
Balder/use highlight terms

### PR DESCRIPTION
**Pull request by baldersheim**
@geirst PR

### CODE REVIEW

File: client/go/internal/admin/jvm/application_container.go [hunk 1]
```diff
@@ -118,6 +118,7 @@ func (a *ApplicationContainer) configureMemory(qc *QrStartConfig) {
 	opts.AddOption(fmt.Sprintf("-Xmx%dm", jvm_heapsize))
 	opts.AddOption(fmt.Sprintf("-XX:ThreadStackSize=%d", jvm_stacksize))
 	opts.AddOption(fmt.Sprintf("-XX:MaxDirectMemorySize=%dm", maxDirectMemorySize))
+	opts.AddOption(fmt.Sprintf("-XX:Tier4CompileThreshold=1000000"))
```
Comment (geirst) [inline client/go/internal/admin/jvm/application_container.go:121]: Was this change intentional?

File: streamingvisitors/src/vespa/searchvisitor/searchvisitor.cpp [hunk 2]
```diff
@@ -443,6 +445,12 @@ SearchVisitor::init(const Parameters & params)
                             i, j, string(prop.key(j)).c_str(), string(prop.value(j)).c_str());
                         _rankController.getFeatureOverrides().add(prop.key(j), prop.value(j));
                     }
+                } else if (prop.name() == "highlightterms") { // pick up feature overrides
```
Comment (geirst) [inline streamingvisitors/src/vespa/searchvisitor/searchvisitor.cpp:448]: Typo, change to: "pick up highlight terms"

**Review by geirst — DISMISSED** [review]
👍

File: client/go/internal/admin/jvm/application_container.go (hunk 1 above)
Comment (baldersheim) [reply client/go/internal/admin/jvm/application_container.go]: No, leftover from experiments. Removed, Thanks.

File: streamingvisitors/src/vespa/searchvisitor/searchvisitor.cpp (hunk 2 above)
Comment (baldersheim) [reply streamingvisitors/src/vespa/searchvisitor/searchvisitor.cpp]: Comment removed.

**Review by geirst — APPROVED** [review]
👍
